doc/contributing.md

```diff
@@ -15,7 +15,7 @@ There is no wrongdoing, just wrong restraint.
 You're in luck. There's various sources for tasks:
 
  - have a look at the [issue tracker](https://github.com/sfttech/openage/issues), especially at the issues labeled [easy](https://github.com/SFTtech/openage/labels/easy) or [just do it](https://github.com/SFTtech/openage/labels/just%20do%20it)
- - use [your favourite code searching tool](https://github.com/ggreer/the_silver_searcher) to find `TODO` messages (there's quite a bunch of them)
+ - use [your favorite code searching tool](https://github.com/ggreer/the_silver_searcher) to find `TODO` messages (there's quite a bunch of them)
```

Comment:
Nasty nasty US English ;)

Is there actually any decision between one or the other?